examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>Null multinomial model -&gt; Vanilla / Plain multinomial model?</p><p> </p><p>Null has a certain connotation in the stats world</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<blockquote>Null has a certain connotation in the stats world</blockquote><p> </p><p>Yeah, I was torn about some of the model "names" (along with the naming of other python variables, too).</p><p> </p><p>I'm down for <code>model_multinomial</code> , <code>model_dm_explicit</code>,  and <code>model_dm_marginalized</code>. Do those seem good to you?</p>

Reply:
Sounds good. You can also drop the explicit and just call it dm or call it latent dm